styles/style.css

```diff
@@ -6,3 +6,183 @@ Green: #00B172
+    margin: 0px;
+
+}
+header{
+    
+    margin: 0;
+    position: fixed;
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+}
+.nav-bar{
+    position: fixed;
+    display: flex;
+    background-color: white;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+}
+.nav-bar img{
+    max-width: 200px;
+    padding: 20px 30px;
+}
+.nav-bar ul{
+    display: flex;
+    justify-content: center;
+    width: 30%;
+    justify-content: space-evenly;
+    text-align: center;
+    margin-top: 40px;
+    margin-right: 40px;
+}
+.nav-bar li{
+    list-style-type: none;
+}
+.nav-bar a{
+    text-decoration: none;
+    color: black;
+}
+
+.section-1-div{
+    position: static;
+    margin-top: 0 ;
+}
+.section-1-div img{
+    width: 100%;
+    margin-top: 100px;
+    position: relative;
+    
+}
+#text-section-1{
+    font-size: 2rem;
+    color: white;
+    position: absolute;
+    top: 70%;
+    left: 50%;
+  transform: translate(-50%, -50%);
+  text-align: center;
+}
+
+.section-1-div h1{
+    font-size: 4rem;
+  padding-bottom: 50px;
+  
+}
+.section-1-div p{
+    width: 730px;
+    font-weight: 100;
+}
+.section-2{
+    padding-bottom: 40px;
+}
+.section-2 h3{
+    padding: 30px 10px 10px 10px;
+    font-size: 3rem;
+    text-align: center;
+   
+}
+.section-2 hr{
+    border: 2px solid #00B172;
+    width: 450px;
+    margin: auto;
+}
+.sub-section2{
+    display: flex;
+    padding: 50px;
+    justify-content: space-evenly;
+}
+.sub-section2 img{
+    padding-top: 40px;
+    width: 150px;
+    padding-bottom: 30px;
+}
+.article{
+    font-size: 2rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+   padding: 20px;
+   height: 200px;
+}
+.article img{
+    justify-content: space-evenly;
+    margin: auto;
+
+}
+.article h4{
+    text-align: center;
+    margin: auto;
+    padding-bottom: 20px;
+    color: #00B172;
+
+}
+.article p{
+    text-align: center;
+    margin: auto;
+
+}
+.section-3{
+    display: flex;
+    background-color: #00B172;
+    color: white;
+    margin: auto;
+    justify-content: space-evenly;
+    align-items: center;
+    font-size: 1.5rem;
+    
+}
+#section-3-div-1{
+    display: flex;
+    flex-direction: column;
+    padding: 10px 30px 10px 30px;
+    width: 30%;
+}
+
+#section-3-div-1 h3{
+    font-size: 1.5rem;
+    padding-bottom: 10px;
+}
+#section-3-div-1 hr{
+    border: 1px solid white;
+    margin-bottom: 30px;
+    width: 85%;
+};
+#section-3-div-1 h4{
+    font-size: 0.80rem;
+    padding-bottom: 10px;
+    padding-top: 10px;
+}
+#section-3-div-1 p{
+    font-size: 1rem;
+    font-weight: 100;
+    padding-bottom: 10px;
+    padding-top: 10px;
+    line-height: 1.5rem;
+    margin-bottom: 30px
+}
+#spotify-logo-white{
+    height: 100px;
+    position: absolute;
+    left: 50%;
+    top: 260%;
+    margin: auto;
+}
+#spotify-app-pic{
+    height: 500px;
+    padding: 70px 00px 40px 50px;
+}
+
+/*
```

Comment:
What is the function of this here? :)

Reply:
A test as well which I forgot to delete. Next time I will review my code better.
Thank you very much for the feedback !